samples/client/petstore/php/SwaggerClient-php/phpunit.xml.dist

```diff
@@ -14,8 +14,8 @@
 
     <filter>
         <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">./lib/Api</directory>
-            <directory suffix=".php">./lib/Model</directory>
+            <directory suffix=".php">./lib\Api</directory>
```

Comment:
@dkarlovi ok. Thanks for the clarification.

If no further feedback from anyone, I'll merge it later today.